core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -516,7 +516,23 @@
   TSERV_SUMMARY_RETRIEVAL_THREADS("tserver.summary.retrieval.threads", "10", PropertyType.COUNT,
       "The number of threads on each tablet server available to retrieve"
           + " summary data, that is not currently in cache, from RFiles."),
-
+  TSERV_COMPRESSOR_FACTORY("tserver.compressor.factory.class",
+      "org.apache.accumulo.core.file.rfile.bcfile.codec.CompressorFactory", PropertyType.CLASSNAME,
+      "Tablet Server configuration for the compressor factory that will be used when requesting compressors."),
+  TSERV_COMPRESSOR_IN_BUFFER("tserver.compressor.factory.input.buffer.size", "1K",
+      PropertyType.MEMORY,
+      "Tablet Server configuration for the compressor factory that adjusts the input buffer size. Zero uses the full compression block size."),
+  TSERV_COMPRESSOR_OUT_BUFFER("tserver.compressor.factory.output.buffer.size", "1K",
+      PropertyType.MEMORY,
+      "Tablet Server configuration for the compressor factory that adjusts the output buffer size. Default uses the full compression block size."),
+  TSERV_COMPRESSOR_POOL_IDLE("tserver.compressor.pool.max.idle", "25", PropertyType.COUNT,
+      "Tablet Server configuration to contrain the maximum number of idle compressors within the pool"),
+  TSERV_COMPRESSOR_POOL_IDLE_SWEEP_TIME("tserver.compressor.pool.max.idle.sweep.time", "0ms",
```

Comment:
I think these are options for a specific plugin.  If someone configures another plugin class, then these options may not be relevant.  Could possibly structure props like :

```
tserver.compressor.factory.class=<class>
tserver.compressor.factory.opts.<key1>=<value1>
tserver.compressor.factory.opts.<key2>=<value2>
```

And the plugin would be given a map containing `<key1>=<value1>,<key2>=<value2>`